pkg/gateway/client/mcpoauthtoken.go

```diff
@@ -231,5 +231,5 @@ func (c *Client) decryptMCPOAuthToken(ctx context.Context, token *types.MCPOAuth
 }
 
 func mcpOAuthTokenCtx(token *types.MCPOAuthToken) value.Context {
-	return value.DefaultContext(fmt.Sprintf("%s/%s", mcpOAuthTokenGroupResource.String(), token.MCPServerInstance))
+	return value.DefaultContext(fmt.Sprintf("%s/%s", mcpOAuthTokenGroupResource.String(), token.MCPID))
```

Comment:
This means that none of the existing tokens in main will work after this goes in. That's fine, but there may be a few issues to resolve once this goes in.

Reply:
Yeah I got permission from Craig to just fully wipe Main and start over there. Though I don't think I have the permissions to do that on my own.